frontends/web/src/routes/device/bitbox01/settings/components/randomnumber.jsx

```diff
@@ -44,18 +44,20 @@ export default class RandomNumber extends Component {
     abort = () => {
         this.setState({
             active: false,
-            number: undefined,
+            number: '',
         });
     }
 
-    render({ t }, { number, active }) {
+    render() {
+        const { t } = this.props;
+        const { number, active } = this.state;
         return (
             <div>
                 <SettingsButton onClick={this.getRandomNumber}>
                     {t('random.button')}
                 </SettingsButton>
                 {
-                    active && number ? (
+                    active && number !== '' ? (
```

Comment:
@x1ddos ack?
alternative: keep as is and add ts-ignore which do you prefer?

Reply:
I discussed with @x1ddos and the other changes are ok, but this one actually changes the logic and here it would be better to just ts-ignore this line

Reply:
@BrianCraig please ts-ignore this line and you'll need to add another ignore, see:

```diff
diff --git a/frontends/web/src/routes/device/bitbox01/settings/components/randomnumber.jsx b/frontends/web/src/routes/device/bitbox01/settings/components/randomnumber.jsx
index 5222eda4..4907d247 100644
--- a/frontends/web/src/routes/device/bitbox01/settings/components/randomnumber.jsx
+++ b/frontends/web/src/routes/device/bitbox01/settings/components/randomnumber.jsx
@@ -48,19 +48,25 @@ export default class RandomNumber extends Component {
         });
     }
 
-    render({ t }, { number, active }) {
+    render() {
+        const { t } = this.props;
+        const { number, active } = this.state;
         return (
             <div>
                 <SettingsButton onClick={this.getRandomNumber}>
                     {t('random.button')}
                 </SettingsButton>
                 {
+                    // @ts-ignore Object is possibly 'undefined'.
                     active && number ? (
                         <Dialog title="Generate Random Number" onClose={this.abort}>
                             <div className="columnsContainer half">
                                 <div className="columns">
                                     <div className="column">
-                                        <p>{t('random.description', { bits: number.length * 4 })}</p>
+                                        <p>{t('random.description', {
+                                            // @ts-ignore
+                                            bits: number.length * 4
+                                        })}</p>
                                         <CopyableInput value={number} flexibleHeight />
                                     </div>
                                 </div>
```

Reply:
and please rebase on top of master

Reply:
so the actual errors in tslint look like this:

```
ERROR in ./src/routes/device/bitbox01/bitbox01.jsx(151,14):
TS2678: Type 'string' is not comparable to type 'null'.
```

in other places, it looks ok but in this one place here, the condition is now different because the `number` can still be set to `null` from the backend: 

```
active && number !== '' ? ...
```

so, that's the reason in https://github.com/digitalbitbox/bitbox-wallet-app/pull/1485/files#r758582958.